### PR TITLE
DOCSUP-18084: Edit and translate to Russian

### PR DESCRIPTION
**Pull request by andrc1901**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #31148, #30949, #30799

### CODE REVIEW

File: docs/ru/interfaces/formats.md [hunk 1]
```diff
@@ -129,6 +129,9 @@ world
 
 Каждый элемент структуры типа [Nested](../sql-reference/data-types/nested-data-structures/nested.md) представляется как отдельный массив.
 
+Значения перечисления, в качестве входных данных, могут быть представлены как имя или как идентификаторы. Сначала мы пытаемся сопоставить входное значение с именем перечисления. В случае неудачи и при условии, что входное значение является числом, мы пытаемся сопоставить это число с идентификатором перечисления.
```
Comment (olgarev) [inline docs/ru/interfaces/formats.md:132]:
```suggestion
Входящие параметры типа "перечисление" (`ENUM`) могут передаваться в виде значений или порядковых номеров. Сначала переданное значение будет сопоставляться с элементами перечисления. Если совпадение не будет найдено и при этом переданное значение является числом, оно будет трактоваться как порядковый номер в перечислении.
```

File: docs/ru/interfaces/formats.md [hunk 2]
```diff
@@ -129,6 +129,9 @@ world
...
+Если входные данные содержат только идентификаторы перечисления, рекомендуется включить настройку [input_format_tsv_enum_as_number](../operations/settings/settings.md#settings-input_format_tsv_enum_as_number) для оптимизации парсинга перечисления.
```
Comment (olgarev) [inline docs/ru/interfaces/formats.md:133]:
```suggestion
Если входящие параметры типа `ENUM` содержат только порядковые номера, рекомендуется включить настройку [input_format_tsv_enum_as_number](../operations/settings/settings.md#settings-input_format_tsv_enum_as_number) для ускорения парсинга.
```

File: docs/ru/interfaces/formats.md [hunk 3]
```diff
@@ -362,6 +365,9 @@ $ clickhouse-client --format_csv_delimiter="|" --query="INSERT INTO test.csv FOR
 
 Если установлена настройка [input_format_defaults_for_omitted_fields = 1](../operations/settings/settings.md#session_settings-input_format_defaults_for_omitted_fields) и тип столбца не `Nullable(T)`, то пустые значения без кавычек заменяются значениями по умолчанию для типа данных столбца.
 
+Значения перечисления, в качестве входных данных, могут быть представлены как имя или как идентификаторы. Сначала мы пытаемся сопоставить входное значение с именем перечисления. В случае неудачи и при условии, что входное значение является числом, мы пытаемся сопоставить это число с идентификатором перечисления.
+Если входные данные содержат только идентификаторы перечисления, рекомендуется включить настройку [input_format_tsv_enum_as_number](../operations/settings/settings.md#settings-input_format_tsv_enum_as_number) для оптимизации парсинга перечисления.
```
Comment (olgarev) [inline docs/ru/interfaces/formats.md:369]:
```suggestion
Входящие параметры типа "перечисление" (`ENUM`) могут передаваться в виде значений или порядковых номеров. Сначала переданное значение будет сопоставляться с элементами перечисления. Если совпадение не будет найдено и при этом переданное значение является числом, оно будет трактоваться как порядковый номер в перечислении.
Если входящие параметры типа `ENUM` содержат только порядковые номера, рекомендуется включить настройку [input_format_tsv_enum_as_number](../operations/settings/settings.md#settings-input_format_tsv_enum_as_number) для ускорения парсинга.
```

File: docs/ru/operations/settings/settings.md [hunk 4]
```diff
@@ -391,12 +391,12 @@ INSERT INTO test VALUES (lower('Hello')), (lower('world')), (lower('INSERT')), (
 
 ## input_format_tsv_enum_as_number {#settings-input_format_tsv_enum_as_number}
 
-Включает или отключает парсинг значений перечислений как идентификаторов перечислений для входного формата TSV.
+При включенном режиме всегда обрабатывайте значения перечисления как идентификаторы перечисления для входного формата TSV. Для оптимизации парсинга, рекомендуется включать этот параметр, если данные содержат только идентификаторы перечисления.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:394]:
```suggestion
Включает или отключает парсинг значений перечислений как порядковых номеров. 

Если режим включен, то во входящих данных в формате `TCV` значения перечисления (тип `ENUM`) всегда трактуются как порядковые номера, а не как элементы перечисления. Эту настройку рекомендуется включать для оптимизации парсинга, если данные типа `ENUM` содержат только порядковые номера, а не сами элементы перечисления.
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -391,12 +391,12 @@ INSERT INTO test VALUES (lower('Hello')), (lower('world')), (lower('INSERT')), (
...
 
 Возможные значения:
 
--   0 — парсинг значений перечисления как значений.
--   1 — парсинг значений перечисления как идентификаторов перечисления.
+-   0 — данные перечисления обработаны как значения или как идентификаторы.
+-   1 — данные перечисления обработаны только как идентификаторы.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:400]:
```suggestion
-   0 — входящие значения типа `ENUM` сначала сопоставляются с элементами перечисления, а если совпадений не найдено, то трактуются как порядковые номера.
-   1 — входящие значения типа `ENUM` сразу трактуются как порядковые номера.

```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -1511,12 +1531,12 @@ SELECT area/period FROM account_orders FORMAT JSON;
 
 ## input_format_csv_enum_as_number {#settings-input_format_csv_enum_as_number}
 
-Включает или отключает парсинг значений перечислений как идентификаторов перечислений для входного формата CSV.
+При включенном режиме всегда обрабатывайте значения перечисления как идентификаторы перечисления для входного формата CSV. Для оптимизации парсинга, рекомендуется включать этот параметр, если данные содержат только идентификаторы перечисления.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:1534]:
```suggestion
Включает или отключает парсинг значений перечислений как порядковых номеров. 
Если режим включен, то во входящих данных в формате `CSV` значения перечисления (тип `ENUM`) всегда трактуются как порядковые номера, а не как элементы перечисления. Эту настройку рекомендуется включать для оптимизации парсинга, если данные типа `ENUM` содержат только порядковые номера, а не сами элементы перечисления.
```

File: docs/ru/operations/settings/settings.md [hunk 7]
```diff
@@ -1511,12 +1531,12 @@ SELECT area/period FROM account_orders FORMAT JSON;
...
 
 Возможные значения:
 
--   0 — парсинг значений перечисления как значений.
--   1 — парсинг значений перечисления как идентификаторов перечисления.
+-   0 — данные перечисления обработаны как значения или как идентификаторы.
+-   1 — данные перечисления обработаны только как идентификаторы.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:1539]:
```suggestion
-   0 — входящие значения типа `ENUM` сначала сопоставляются с элементами перечисления, а если совпадений не найдено, то трактуются как порядковые номера.
-   1 — входящие значения типа `ENUM` сразу трактуются как порядковые номера.
```

**Review by KochetovNicolai — APPROVED** [review]

**Comment by KochetovNicolai** [comment]
@mergifyio up

**Comment by KochetovNicolai** [comment]
@Mergifyio update

**Comment by KochetovNicolai** [comment]
@Mergifyio update

**Pull request merged**